gtk4/src/icon_theme.rs

```diff
@@ -1,13 +0,0 @@
-// Take a look at the license at the top of the repository in the LICENSE file.
-
-use crate::IconTheme;
-use glib::translate::*;
-
-impl IconTheme {
-    #[doc(alias = "gtk_icon_theme_set_resource_path")]
-    pub fn set_resource_path(&self, path: &str) {
```

Comment:
Why did this disappear?

Reply:
See `gtk: generate and fix set_resource_path()`. I am not even clear whether the current binding actually work, since the strv argument may not be NULL-terminated.